package.json

```diff
@@ -41,8 +41,10 @@
     "dotenv-webpack": "^1.5.4",
     "extract-text-webpack-plugin": "^3.0.0",
     "graphql": "^14.0.0",
```

Comment:
I'm unsure if `"graphql"` is allowed to be a regular dep if it's a peer dependencies. I guess it's fine if npm isn't warning us about this.

Reply:
We do the same for `webpack` so I think it's 👌